tests/devices/test_default_mixed.py

```diff
@@ -255,9 +255,9 @@ class TestKrausOps:
     """Unit tests for the method `_get_kraus_ops()`"""
 
     unitary_ops = [
-        (PauliX, np.array([[0, 1], [1, 0]])),
-        (Hadamard, np.array([[INV_SQRT2, INV_SQRT2], [INV_SQRT2, -INV_SQRT2]])),
-        (CNOT, np.array([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]])),
+        (PauliX(wires=0), np.array([[0, 1], [1, 0]])),
```

Comment:
wait, how come this has changed?

Reply:
Because then we can just do `matrix()` below, which is much more convenient than `compute_matrix`